modules/nw-sriov-about-qinq.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// 
+// 
+// 
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-about-qinq-support_{context}"]
+= About 802.1q-in-802.1q support
+
+QinQ, formally known as 802.1q-in-802.1q, is a networking technique defined by IEEE 802.1ad. IEEE 802.1ad, extends the IEEE 802.1Q-1998 standard, enriches VLAN capabilities by introducing an additional 802.1Q tag to packets already tagged with 802.1Q. This method is also referred to as VLAN stacking or double VLAN.
+
+In traditional VLAN setups, frames typically contain a single VLAN tag, such as VLAN-100, alongside other metadata such as Quality of Service (QoS) bits and protocol information. QinQ introduces a second VLAN tag, where the service provider designates the outer tag for their use, offering them flexibility, while the inner tag remains dedicated to the customer's VLAN.
+
+By employing double VLAN tagging, QinQ facilitates the creation of nested VLANs, enabling finer segmentation and traffic isolation within a network environment. This approach is particularly valuable in service provider networks, where VLAN-based services need to be delivered to multiple customers over a shared infrastructure, all while ensuring traffic separation and isolation.
+
+When two VLAN tags are present in a packet, the outer VLAN tag can be either 802.1q or 802.1ad. The inner VLAN tag must always be 802.1q.
+
+The {product-title} SR-IOV solution already supports 802.1q/802.1q, by seting a VLAN in the `SriovNetwork` object and configuring a additional VLAN interface internally in the pod. 
```

Comment:
I would say instead that our solution supports setting the vlan protocol on the sriovNetwork which will be used by the VF to set the vlan tag (outer tag). Pods might use the vlan cni (referrence to the vlan cni section) to configure the inner tag. Something like that.